File: onyx-database/src/main/java/com/onyx/exception/NoResultsException.java
package com.onyx.exception;

/**
 * Created by timothy.osborn on 11/30/14.
 *
 * Entity does not return results when expected
 */
public class NoResultsException extends EntityException {

    public NoResultsException()
    {
        super();
    }

    @SuppressWarnings("unused")
    public NoResultsException(Throwable e)
    {
        super(e);
    }
}

File: onyx-database/src/main/kotlin/com/onyx/exception/NoResultsException.kt
package com.onyx.exception

/**
 * Created by timothy.osborn on 11/30/14.
 *
 * Entity does not return results when expected
 */
class NoResultsException : OnyxException {

    constructor() : super()

    constructor(e: Throwable) : super(e)
}


